lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -3418,11 +3436,11 @@ DefMacroI('\floatpagefraction', undef, "0.25");
 NewCounter('dbltopnumber');
 DefMacroI('\dbltopfraction',       undef, "0.7");
 DefMacroI('\dblfloatpagefraction', undef, "0.25");
-DefRegister('\floatsep'         => Glue(0));
-DefRegister('\textfloatsep'     => Glue(0));
-DefRegister('\intextsep'        => Glue(0));
-DefRegister('\dblfloatsep'      => Glue(0));
-DefRegister('\dbltextfloatsep'  => Glue(0));
+DefRegister('\floatsep'         => Glue('12.0pt plus 2.0pt minus 2.0pt'));
+DefRegister('\textfloatsep'     => Glue('20.0pt plus 2.0pt minus 4.0pt'));
```

Comment:
there was a macro parsing these by looking for `\def\foo #1 plus #2 minus #3 {work with #1 #2 #3} `, which was failing, so I chose to add the glue values for the various seps, as pdflatex reported them.